ui-tests/src/test/java/org/openmrs/reference/VisitNoteTest.java

```diff
@@ -51,25 +49,31 @@ public void VisitNoteTest() throws Exception {
         visitNotePage.addSecondaryDiagnosis(DIAGNOSIS_SECONDARY);
         visitNotePage.addNote("This is a note");
         patientDashboardPage = visitNotePage.save();
-        assertEquals(DIAGNOSIS_PRIMARY, visitNotePage.primaryDiagnosis());
-        assertEquals(DIAGNOSIS_SECONDARY, visitNotePage.secondaryDiagnosis());
-
-        PatientVisitsDashboardPage patientVisitsDashboardPage = patientDashboardPage.goToRecentVisits();
-
+        patientDashboardPage.waitForPage();
+        patientVisitsDashboardPage = patientDashboardPage.goToRecentVisits();
         EditVisitNotePage editVisitNotePage = patientVisitsDashboardPage.goToEditVisitNote();
-
-        editVisitNotePage.deleteDiagnosis();
+        
+        //Editing visit Note 
```

Comment:
Should we maybe break out editing a note as a separate test?

Reply:
We are trying to make them independent, breaking the editing note would mean some tests will be dependent on each other